### PR TITLE
chore: update kitex blog [2021-11-24]

### PR DESCRIPTION
**Pull request by GuangmingLuo**
Ref: [RPC 框架 Kitex 实践入门：性能测试指南](https://www.cloudwego.io/zh/blog/2021/11/24/rpc-%E6%A1%86%E6%9E%B6-kitex-%E5%AE%9E%E8%B7%B5%E5%85%A5%E9%97%A8%E6%80%A7%E8%83%BD%E6%B5%8B%E8%AF%95%E6%8C%87%E5%8D%97/)

changes:
1. fix one paragraph wrong layout
2. add intro for [The Characteristics of Microservice Scenario]
3. fix typos
4. adjust some wrong or improper words with better ones
5. add blank Lines for paragraphs

### CODE REVIEW

File: content/en/blog/news/kitex_performance_testing/index.md [hunk 1]
```diff
@@ -1,85 +1,128 @@
...
+Therefore, the following will first introduce the characteristics of microservices, so that developers can understand Kitex's design thinking in depth.
 
-* **Size of Data Package**  
-Although the size of transmitted data packages depends on the actual business scenario, the internal statistics of ByteDance found that most online requests are small packages (<2KB). So we focused on optimizing the performance in the small data package scenarios while taking the large package scenarios into account.  
+* **RPC Communication Model**
+An RPC call often requires multiple microservices to collaborate, and downstream services have their own dependencies,  so the entire call chain will be a complex network structure.
 
-## Stress Test for Microservice Scenarios  
+* **Complex Call Chain**
+The communication between microservices is usually based on PingPong model. So, in addition to the conventional throughput performance index, developers also need to consider the average latency of each RPC.
```
Comment (yccpt) [inline content/en/blog/news/kitex_performance_testing/index.md:29]: An RPC call often requires multiple microservices to collaborate, and downstream services have their own dependencies,  so the entire call chain will be a complex network structure.

File: content/en/blog/news/kitex_performance_testing/index.md [hunk 2]
```diff
@@ -1,85 +1,128 @@
...
+Kitex was born in ByteDance's large-scale microservices architecture practice. The scenario it is aimed at is naturally a microservices scenario.
+Therefore, the following will first introduce the characteristics of microservices, so that developers can understand Kitex's design thinking in depth.
 
-* **Size of Data Package**  
-Although the size of transmitted data packages depends on the actual business scenario, the internal statistics of ByteDance found that most online requests are small packages (<2KB). So we focused on optimizing the performance in the small data package scenarios while taking the large package scenarios into account.  
+* **RPC Communication Model**
+An RPC call often requires multiple microservices to collaborate, and downstream services have their own dependencies,  so the entire call chain will be a complex network structure.
```
Comment (yccpt) [inline content/en/blog/news/kitex_performance_testing/index.md:26]: 这里需要换成"The communication between microservices is usually based on PingPong model. So, in addition to the conventional throughput performance index, developers also need to consider the average latency of each RPC."

File: content/en/blog/news/kitex_performance_testing/index.md (hunk 1 above)
Comment (yccpt) [inline content/en/blog/news/kitex_performance_testing/index.md:29]: 29 这里需要换成”An RPC call often requires multiple microservices to collaborate, and downstream services have their own dependencies,  so the entire call chain will be a complex network structure.“

File: content/en/blog/news/kitex_performance_testing/index.md [hunk 3]
```diff
@@ -1,85 +1,128 @@
 ---
 date: 2021-11-24
-title: "Getting Started With RPC Practices: Performance Testing Guide"
-linkTitle: "Getting Started With RPC Practices: Performance Testing Guide"
+title: "Getting Started With Kitex's Practices: Performance Testing Guide"
```
Comment (yccpt) [inline content/en/blog/news/kitex_performance_testing/index.md:3]: practice作为实践的意思 不可数

**Review by joway — APPROVED** [review]

**Pull request merged**